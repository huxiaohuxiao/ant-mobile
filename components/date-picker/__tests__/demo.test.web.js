import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('date-picker');
