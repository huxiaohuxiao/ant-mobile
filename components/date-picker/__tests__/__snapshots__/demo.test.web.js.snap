// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/date-picker/demo/basic.md correctly 1`] = `
<div>
  <div
    class="am-list date-picker-list"
    style="background-color:white;"
  >
    <div
      class="am-list-body"
    >
      <div>
        <div
          class="am-list-item am-list-item-middle"
        >
          <div
            class="am-list-line"
          >
            <div
              class="am-list-content"
            >
              日期(CST)
            </div>
            <div
              class="am-list-extra"
            >
              可选,小于结束日期
            </div>
            <div
              aria-hidden="true"
              class="am-list-arrow am-list-arrow-horizontal"
            />
          </div>
          <div
            class="am-list-ripple"
            style="display:none;"
          />
        </div>
      </div>
      <div>
        <div
          class="am-list-item am-list-item-middle"
        >
          <div
            class="am-list-line"
          >
            <div
              class="am-list-content"
            >
              时间(local time)
            </div>
            <div
              class="am-list-extra"
            >
              请选择
            </div>
            <div
              aria-hidden="true"
              class="am-list-arrow am-list-arrow-horizontal"
            />
          </div>
          <div
            class="am-list-ripple"
            style="display:none;"
          />
        </div>
      </div>
      <div>
        <div
          class="am-list-item am-list-item-middle"
        >
          <div
            class="am-list-line"
          >
            <div
              class="am-list-content"
            >
              时间(CST)，限定上下限
            </div>
            <div
              class="am-list-extra"
            >
              08:00
            </div>
            <div
              aria-hidden="true"
              class="am-list-arrow am-list-arrow-horizontal"
            />
          </div>
          <div
            class="am-list-ripple"
            style="display:none;"
          />
        </div>
      </div>
      <div>
        <div
          class="am-list-item am-list-item-middle"
        >
          <div
            class="am-list-line"
          >
            <div
              class="am-list-content"
            >
              日期+时间
            </div>
            <div
              class="am-list-extra"
            >
              2016-11-22 08:00
            </div>
            <div
              aria-hidden="true"
              class="am-list-arrow am-list-arrow-horizontal"
            />
          </div>
          <div
            class="am-list-ripple"
            style="display:none;"
          />
        </div>
      </div>
      <div>
        <div
          class="am-list-item am-list-item-middle"
        >
          <div
            class="am-list-line"
          >
            <div
              class="am-list-content"
            >
              time(utc, UK time)
            </div>
            <div
              class="am-list-extra"
            >
              00:00 +00:00
            </div>
            <div
              aria-hidden="true"
              class="am-list-arrow am-list-arrow-horizontal"
            />
          </div>
          <div
            class="am-list-ripple"
            style="display:none;"
          />
        </div>
      </div>
      <div
        class="am-list-item am-list-item-middle"
      >
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            <div>
              自定义控制显示/隐藏的元素
            </div>
          </div>
          <div
            class="am-list-extra"
          />
        </div>
        <div
          class="am-list-ripple"
          style="display:none;"
        />
      </div>
      <div>
        <div
          style="background-color:#fff;height:0.9rem;line-height:0.9rem;padding:0 0.3rem;"
        >
          时间选择(自定义 children)
          <span
            style="float:right;color:#888;"
          >
            请选择(可选)
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
